modules/apps/callbacks/transfer_test.go

```diff
@@ -0,0 +1,217 @@
+package ibccallbacks_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v7/modules/apps/callbacks/types"
+	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+)
+
+const (
+	callbackAddr = "cosmos1q4hx350dh0843y34n0vm4lfj6eh5qz4sqfrnq0"
```

Comment:
any reason we can't use the ibctesting `TestAccAddress` instead?

Reply:
done